projects/managed-hubs/support.md

```diff
@@ -107,6 +107,14 @@ Hub Administrators
   
 ```
 
+### Who are our Support Team?
+
+The Support Steward role rotates through the below Support Team members.
+
+```{eval-rst}
```

Comment:
Not a big deal but why not just use MyST markdown for these instead of eval-rst ?

Reply:
I was copying from the JupyterHub team compass docs, so it wasn't so much a "decision I made" as a "I have no idea how to do this so I'll copy an example I know works" scenario. I don't have enough working knowledge of MyST markdown to make on the fly replacements yet.

Reply:
Mind if I push the equivalent to show how it is structured?

Also this makes me wonder - how can we expose Myst documentation to the team in a more accessible manner? We use MyST throughout our docs so I wanna make sure it isn't a barrier for others

Reply:
Please feel free! I would love to be able to learn more, but right now I just have to go through others' examples and figure it out on the way.